sound/soc/sof/topology.c

```diff
@@ -725,6 +735,13 @@ static const struct sof_topology_token core_tokens[] = {
 		offsetof(struct sof_ipc_comp, core), 0},
 };
 
+/* Component extended tokens */
+static const struct sof_topology_token comp_ext_tokens[] = {
+	{SOF_TKN_COMP_UUID,
+		SND_SOC_TPLG_TUPLE_TYPE_UUID, get_token_uuid,
+		offsetof(struct sof_ipc_comp_new_ext, uuid), 0},
+};
+
```

Comment:
I'm thinking about TKN name, shouldn't it be rather something more generic and related with struct name, like `SOF_TKN_COMP_NEW_EXT` ?

Reply:
not catch, what change do you propose?

Reply:
change `SOF_TKN_COMP_UUID` to `SOF_TKN_COMP_NEW_EXT` because of close relation with `struct sof_ipc_comp_new_ext` and possibility to extended in future

Reply:
the tuple can not be a struct, we need to add more tuples e.g. SOF_TKN_COMP_INDEX to this comp_ext_tokens[] array in future.

Reply:
ok